linkis-dist/package/sbin/kill-process-by-pid.sh

```diff
@@ -14,68 +14,59 @@
 # limitations under the License.
 #
 
-function list_descendants ()
-{
-  ppid=$1
-  local children=$(pgrep -P ${ppid})
+cd `dirname $0`
+cd ..
+INSTALL_HOME=`pwd`
+# set LINKIS_HOME
+if [ "$LINKIS_HOME" = "" ]; then
+  export LINKIS_HOME=$INSTALL_HOME
+fi
+if [ "$LINKIS_LOG_DIR" = "" ]; then
+  export LINKIS_LOG_DIR="/data/bdp/logs/linkis"
```

Comment:
Is it better to change it to `LINKIS_LOG_DIR="$LINKIS_HOME/logs" `

Reply:
ok,Thanks